Hymns/Tags/TagSheetView.swift

```diff
@@ -0,0 +1,62 @@
+import SwiftUI
+import Resolver
+
+struct TagSheetView: View {
+
+    @ObservedObject private var viewModel: TagSheetViewModel
+    @Environment(\.presentationMode) var presentationMode
+    @State private var tagName = ""
+    @State private var tagColor = TagColor.none
+    var sheet: Binding<DisplayHymnSheet?>
+
+    init(viewModel: TagSheetViewModel, sheet: Binding<DisplayHymnSheet?>) {
+        self.viewModel = viewModel
+        self.sheet = sheet
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("Name your tag").font(.body).fontWeight(.bold)
+            TextField("Label it however you like", text: self.$tagName)
+            Divider()
+            ColorSelectorView(tagColor: self.$tagColor).padding(.vertical)
+            if !self.viewModel.tags.isEmpty {
+                Section {
+                    Text("Tags").font(.body).fontWeight(.bold)
+                    HStack {
+                        ForEach(self.viewModel.tags, id: \.self) { tag in
+                            Button(action: {
+                                //TODO FIX Delete is deleting all the tags because the button when clicked is clicking all of them.
+                                self.viewModel.deleteTag(tagTitle: tag.title, tagColor: .blue)
+                            }, label: {
+                                HStack {
+                                    Text(tag.title).font(.body).fontWeight(.bold)
+                                    Image(systemName: "xmark.circle")
+                                }
+                            })
+                                .padding(10)
+                                .foregroundColor(tag.color.foreground)
+                                .background(tag.color.background)
+                                .cornerRadius(20)
+                                .lineLimit(1)
```

Comment:
Why line limit 1? What happens if someone enters a really long tag?

Reply:
It applies to the tag. If it's a long tag it just gets .... 
The reason is otherwise it looks pretty janky. Here is it without linelimit 1 
![Simulator Screen Shot - iPhone 8 - 2020-06-13 at 10 51 59](https://user-images.githubusercontent.com/15247726/84573082-062a6d00-ad64-11ea-8bf9-b7248212ce67.png)